problemsets-mat223/modules/module4-exercises.tex

```diff
@@ -181,5 +181,67 @@
 				real numbers.
 			\end{enumerate}
 		\end{solution}
+	
+        	\prob Let $\vec u, \vec v \in \R^{n}$. In this problem, we will prove
+		$\Abs{\vec u \cdot \vec v}\leq \Norm{\vec u}\Norm{\vec v}$. This is called
+		the \emph{Cauchy-Schwarz inequality}.
+		\begin{enumerate}
+			\item Assuming the geometric definition
+				$\vec u \cdot \vec v = \Norm{\vec u}\Norm{\vec v}\cos \theta$
+				where $\theta$ is the angle between $\vec u$ and $\vec v$, prove
+				the \emph{Cauchy-Schwarz inequality}.
+
+			\item The \emph{Cauchy-Schwartz inequality} can also be proved using
+				\emph{only} the algebraic definition of the dot product. Keep in
+				mind the following facts which come from the algebraic definition:
+
```

Comment:
Where did the list of facts go?

Reply:
facts such as:  `$\Norm{\vec a}^2 = \vec a \cdot \vec a`?

Reply:
Yes, and distributivity. You have a colon which introduces nothing....I expected the facts to follow.